src/templates/React InstantSearch/package.json

```diff
@@ -6,29 +6,40 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "lint": "eslint .",
-    "lint:fix": "npm run lint -- --fix"
+    "build:styles": "tailwind build src/index.css -o public/tailwind.css",
```

Comment:
does this mean the styles don't watch? and you need to run this every time you change some css?

Reply:
No, it only builds Tailwind utilities (`text-center`, etc.) when you run `yarn start` or `yarn build`. You can see the result in the sandbox and fetch the branch to try it locally.